### PR TITLE
golangci-lint cleanup

### PR DESCRIPTION
**Pull request by baude**
a PR slipped through without running the new linter.  this cleans things
up for the master branch.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: libpod/oci_internal_linux.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"bufio"
 	"bytes"
 	"fmt"
+	"github.com/containers/libpod/pkg/errorhandling"
```
Comment (mheon) [inline libpod/oci_internal_linux.go]: Import ordering

File: libpod/oci_internal_linux.go [hunk 2]
```diff
@@ -273,7 +274,7 @@ func (r *OCIRuntime) sharedConmonArgs(ctr *Container, cuuid, bundlePath, pidPath
 		logDriver = JournaldLogging
 	case JSONLogging:
 		fallthrough
-	default:
+	default: //nolint-stylecheck)
```
Comment (mheon) [inline libpod/oci_internal_linux.go]: Trailing closeparen?

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Pull request merged**